src/errors.rs

```diff
@@ -1,64 +1,79 @@
-use crate::tenant_security_client::errors::TenantSecurityError;
+pub use crate::tenant_security_client::errors::{
+    KmsError, SecurityEventError, ServiceError, TenantSecretError, TenantSecurityProxyError,
+};
 use crate::vector::crypto::{
     DecryptError as VectorDecryptError, EncryptError as VectorEncryptError,
 };
 
 /// Errors related to IronCore Alloy SDK
-#[derive(Debug, uniffi::Error, PartialEq, Eq)]
-#[uniffi(flat_error)]
+#[derive(Debug, uniffi::Error, PartialEq, Eq, Clone)]
 pub enum AlloyError {
     /// Error while loading configuration.
-    InvalidConfiguration(String),
+    InvalidConfiguration { msg: String },
```

Comment:
Obviously writing `message` instead of `msg` a million times in this churny PR sucks, but `message` is more explicit and straightforward. Don't think anyone is going to struggle to understand `msg` but we generally haven't been dropping vowels in this library so far.